### ISSUE TITLE
Unable to check some rules when css path is incomplete or incorrect

### ISSUE THREAD

**Issue by jlin412** [alecxe/eslint-plugin-protractor#51]
For examples, if I have the following locator and css in the page object:

```
country: this.$homeAddressPendingData.$('[data-e2e="country]')
```

The missing end quote breaks and prints out error message.  I don't think that it is intentional but it is a good feature to have except it doesn't have line number.  

It will be great to have a line number when we see breakage like that.

**Comment by alecxe** [comment]
@jlin412 thanks for the report. Correct me if I am wrong: you are proposing to have a rule that would check if a CSS selector is syntactically valid, right?

**Comment by jlin412** [comment]
that is correct.  It seems that it is already doing with the library but it
is not reporting the line number

**Comment by alecxe** [comment]
@jlin412 gotcha. Currently, I'm skipping invalid CSS selectors, just printing out the parsing error message on the console. And, yes, it does not contain any information about a place where it happened. I'll improve that.

Plus, I'm going to make a new rule which would report invalid CSS selectors separately. Thanks.

**Comment by jlin412** [comment]
COOL!!!

**Comment by alecxe** [comment]
Closed by f0eaf78f0025fa1785e16f954551ee67dcc25246.

I've added a new separate [`no-invalid-selectors`](https://github.com/alecxe/eslint-plugin-protractor/blob/master/docs/rules/no-invalid-selectors.md) rule and removed the console messages from other rules that try to parse CSS selectors. I think this rule is a pretty neat one when used with your IDE of a choice with ESLint enabled on file save.

Thanks for the suggestion. Appreciate it!
